src/common/entity/domain_icon.ts

```diff
@@ -44,6 +47,14 @@ export const domainIcon = (
     case "cover":
       return coverIcon(compareState, stateObj);
 
+    case "device_tracker":
+      switch (stateObj?.attributes.source_type) {
+        case "router":
+          return compareState === "home" ? mdiLanConnect : mdiLanDisconnect;
+        default:
+          return mdiAccount;
```

Comment:
You should remove it from fixed domain icons now
<https://github.com/home-assistant/frontend/blob/dev/src/common/const.ts#L79>

Reply:
thx